### ISSUE TITLE
[Epic] Implement Datawrapper

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#8133]
The ability to easily create and embed data visualizations across our site has been in high demand from staff https://github.com/mozilla/foundation.mozilla.org/issues/6151 & https://github.com/mozilla/foundation.mozilla.org/issues/6645 and [summary of staff survey. ](https://docs.google.com/presentation/d/1YOjMQ5vQf1-scF-bZvkOGVN6Tr5iOpABR_NAKuRiS9Y/edit#slide=id.g25275a8168_0_425)

Ben and @sabrinang looked into different tools and decided on https://www.datawrapper.de/ in https://github.com/mozilla/foundation.mozilla.org/issues/6587. They meet with Datawrapper https://github.com/mozilla/foundation.mozilla.org/issues/6732 and kicked-off the process.

We now have an account with them. They are currently customizing it with our fonts and colors. We should, in theory, be able to now use Datawrapper.

### Next steps
- [x] Review design styles that Datawrapper has implemented [blocked by Datawrapper until ~ Jan 31]
- [ ] Onboard dev(s) and Jen, if needed, to Datawrapper [perhaps next sprint]
- [ ] Add ability to use Datawrapper on our CMS
- [ ] Onboard staff
- [ ] Datawrapper has offered to host an onboarding session 

### Resources from Datawrapper
- https://www.datawrapper.de/training-materials
- https://blog.datawrapper.de/datawrapper-training-materials-for-workshops/
- [Academy](https://academy.datawrapper.de/ ) - place to go to learn more about the tool and the different visualization types
- [Blog](https://blog.datawrapper.de/ ) -  learn about data viz best practices and new features
- [Developer Docs](https://developer.datawrapper.de/docs) - where you can learn more about our [API](https://academy.datawrapper.de/article/225-what-you-can-do-with-our-api-and-how-to-use-it) 

cc @beccaklam

**Comment by kristinashu** [comment]
Thank you @fessehaye for taking this! Let Sabrina or I know if you have any questions. You'll probably want to open a new ticket for the dev work.

**Comment by tbrlpld** [comment]
@kristinashu @sabrinang Is the a particular set of pages on which the Datawrapper block should be available? Or is this something you would want to see site wide?

**Comment by kristinashu** [comment]
@tbrlpld let's start with Article, Banner, Blog, Campaign, and Default. Ty!
